knodle/labeler/labeler.py

```diff
@@ -0,0 +1,19 @@
+from abc import abstractmethod
+
+from .config import LabelerConfig
+
+
+class Labeler:
+    def __init__(
+            self,
+            labeler_config: LabelerConfig = None,
+    ):
+
+        if labeler_config is None:
```

Comment:
`self.labeler_config = LabelerConfig() if labeler_config is None else labeler_config`